recipes/default.rb

```diff
@@ -15,3 +15,166 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+
+node.default['osl-apache']['listen'] = %w(80 443)
+node.default['osl-apache']['worker_mem'] = 215
+
+node.default['osl-postfix']['main']['home_mailbox'] = 'Mail/'
+node.default['osl-postfix']['main']['mailbox_command'] = '/usr/bin/procmail'
+node.default['osl-postfix']['main']['mailbox_size_limit'] = '0'
+node.default['osl-postfix']['main']['message_size_limit'] = '102400000'
+node.default['osl-postfix']['main']['transport_maps'] = 'hash:/etc/postfix/transport'
+
+node.default['postfix']['access']['140.211.166.133'] = 'OK' # smtp2.osuosl.org
+node.default['postfix']['access']['140.211.166.136'] = 'OK' # smtp3.osuosl.org
+node.default['postfix']['access']['140.211.166.137'] = 'OK' # smtp4.osuosl.org
+node.default['postfix']['access']['140.211.166.138'] = 'OK' # smtp1.osuosl.org
+
+include_recipe 'osl-apache'
+include_recipe 'osl-apache::mod_ssl'
+include_recipe 'osl-apache::mod_perl'
+include_recipe 'osl-mysql::client'
+include_recipe 'yum-osuosl'
+include_recipe 'perl'
+
+package %w(request-tracker mutt procmail)
+
+# Root Account
+template '/root/.rtrc' do
+  source 'rt/rtrc.erb'
+  cookbook 'osl-rt'
+  mode '0600'
+  sensitive true
+  variables(root_pass: node['osl-rt']['root-password'])
+end
+
+# Set up user for mail
+user node['osl-rt']['default'] do
+  manage_home true
+end
+
+# User defined Hostalias file in order to patch into the RT site with the RT CLI/procmail
+['root', "/home/#{node['osl-rt']['default']}"].each do |file_path|
+  file "/#{file_path}/.rthost" do
+    content <<~EOF
+      rtlocal localhost
+    EOF
+  end
+end
+
+# Add the RT command to the root user's PATH
+link '/usr/local/sbin/rt' do
+  to '/opt/rt/bin/rt'
+end
+
+# RT Initial Configuration.
+file '/opt/rt/etc/RT_SiteConfig.pm' do
+  # Use the init function in order to generate the perl config file
+  content osl_rt_init_config
+  group 'apache'
+  mode '0640'
+  sensitive true
+  notifies :reload, 'apache2_service[osuosl]'
+end
+
+# Initalize the DB
+execute 'init-db-rt' do
+  command <<-EOC
```

Comment:
Please use `~` globally for these "heredoc" blocks of code. This automatically removes the prepended space when it runs the command and makes it easier to read the output if needed.
```suggestion
  command <<~EOC
```